src/app.js

```diff
@@ -1 +1,50 @@
-console.log('Hallo daar!');
\ No newline at end of file
+//Import axios
+import axios from "axios";
+
+//Maak een async functie waarbij de benodigde informatie wordt opgehaald
+
+async function getCountryDataList() {
+
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all?fields=name,region,flags,population');
```

Comment:
Nice dat je enkel naar binnen haalt wat nodig is. Ook kan je hier de 'subregion' aan toevoegen om onderscheid te kunnen maken tussen Noord en Zuid Amerika. Die vallen onder 'Region' allebei namelijk onder 'Americas'